practice-one/src/index.html

```diff
@@ -47,5 +47,30 @@
         <button class="button" type="button">Request early access</button>
       </div>
     </section>
+
+    <section class="content-summary">
+      <div class="summary">
+        <p class="title title-meaning-of-work">Reimagining what it means to work</p>
+        <p class="description description-meaning-of-work">Teams and communities using Open Enterprise fundamentally unlock a reality of work that 
+          reimagines how people engage in economic opportunity, meeting the demands and expectations of a modern organization.</p>
+      </div>
+      <div class="cards">
+        <figure class="card">
```

Comment:
You should `figcaption` instead of `p`. Pls read more about figure [here](https://www.w3schools.com/tags/tag_figure.asp)